GX-TimeFrequency/src/minimax_grids.F90

```diff
@@ -100,9 +75,9 @@ subroutine gx_minimax_grid(num_points, e_min, e_max, &
     real(kind=dp), dimension(:, :), allocatable       :: mat
     real(kind=dp), dimension(:, :), allocatable       :: tmp_cosft_wt, tmp_cosft_tw
 
-    my_bare_cossine_weights = .false.
-    if (present(bare_cossine_weights)) then
-      if(bare_cossine_weights) my_bare_cossine_weights = .true.
+    my_bare_cosine_weights = .false.
+    if (present(bare_cosine_weights)) then
+       my_bare_cosine_weights = .true.
```

Comment:
`my_bare_cosine_weights = bare_cosine_weights`

else the behaviour is wrong if someone passes 

`bare_cosine_weights = .false.`